src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -104,6 +104,8 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
     const currentXeroOrganization = findCurrentXeroOrganization(tenants, policy?.connections?.xero?.config?.tenantID);
     const shouldShowSynchronizationError = !!synchronizationError;
     const shouldShowReinstallConnectorMenuItem = shouldShowSynchronizationError && connectedIntegration === CONST.POLICY.CONNECTIONS.NAME.QBD;
+    const shouldShowCardReconciliationOption =
```

Comment:
Why do you use these fields to detect if the bank account is settled or not? I think we should use this field
```
private_expensifyCardSettings_{workspaceAccountID}.paymentBankAccountID
```

Reply:
No, It is another flow